mlrun/kfpops.py

```diff
@@ -747,21 +747,32 @@ def format_summary_from_kfp_run(kfp_run, project=None, session=None):
             if error:
                 dag[step]["error"] = error
 
-    short_run = {"graph": dag}
-    short_run["run"] = {
-        k: str(v)
-        for k, v in kfp_run["run"].items()
-        if k
-        in [
-            "id",
-            "name",
-            "status",
-            "error",
-            "created_at",
-            "scheduled_at",
-            "finished_at",
-            "description",
-        ]
+    # align kfp fields to mlrun snake case convention
+    for step in dag:
+        # create snake_case for consistency.
+        # retain the camelCase for compatibility
+        if "startedAt" in dag[step]:
+            dag[step]["started_at"] = dag[step]["startedAt"]
+        if "finishedAt" in dag[step]:
+            dag[step]["finished_at"] = dag[step]["finishedAt"]
```

Comment:
perhaps generalize it for future maintenance?
e.g:
```
for camelcase_field, snakecase_field in [("startedAt", "started_at")...]:
    if camelcase_field in dag[step]:
        dag[step][snakecase_field] = dag[step][camelcase_field]
```

Reply:
just a suggestion:
```
field_mappings = [("startedAt", "started_at"), ("finishedAt", "finished_at")]

for dag_step in dag.values():
    dag_step.update({snakecase_field: dag_step[camelcase_field]
                     for camelcase_field, snakecase_field in field_mappings
                     if camelcase_field in dag_step})
    
```

Reply:
went with tomer's idea